openedx/features/pakx/cms/custom_settings/views.py

```diff
@@ -59,12 +61,23 @@ def post(self, request, course_key_string):
         Save course overview content in model and display updated version of custom settings page
         """
         course_key = CourseKey.from_string(course_key_string)
+
+        course_set = request.POST['course-set']
         course_overview = request.POST['course-overview']
+        card_description = request.POST['card-description']
+        publisher_logo_url = request.POST['publisher-logo-url']
         course_experience = request.POST.get('course_experience', 0)
+
         if course_overview is not None:
             CourseOverviewContent.objects.update_or_create(
```

Comment:
shouldn't we call `get_or_create_course_overview_content` here?

Reply:
get_or_create_course_overview_content does not update the record. can't use that function here.